fontconfig.yaml

```diff
@@ -10,6 +10,13 @@ package:
         - "*"
       attestation: TODO
       license: MIT
+  scriptlets:
```

Comment:
I'm not 100% sure about these scriptlets. I've used examples already in wolfi, are they correct?

Reply:
This one is fine.